Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleSceneView.swift

```diff
@@ -326,7 +346,9 @@ private extension WorldScaleSceneView {
     }
 }
 
+@available(visionOS, unavailable)
 public extension WorldScaleSceneView {
+#if !os(visionOS)
```

Comment:
Are either of these necessary? It seems like neither of them is because `WorldScaleSceneView` is already marked as unavailable on visionOS.

Reply:
If a type is marked as unavailable then all extensions of that type will need that marking as well.

Reply:
Seriously?? That's disappointing. Perhaps there's a valid reason for that, but I can't think of one.